models/product-type/src/attribute-money-type/attribute-money-type-draft/transformers.ts

```diff
@@ -0,0 +1,34 @@
+import { Transformer } from '@commercetools-test-data/core';
+import { TAttributeEnumType } from '../../attribute-enum-type';
+import {
+  TAttributeMoneyTypeDraftGraphql,
+  TAttributeMoneyTypeDraft,
+} from '../types';
+
+const transformers = {
+  default: Transformer<TAttributeMoneyTypeDraft, TAttributeMoneyTypeDraft>(
+    'default',
+    {
+      buildFields: [],
+    }
+  ),
+  rest: Transformer<TAttributeMoneyTypeDraft, TAttributeMoneyTypeDraft>(
+    'rest',
+    {
+      buildFields: [],
+    }
+  ),
+  graphql: Transformer<
+    TAttributeMoneyTypeDraft,
+    TAttributeMoneyTypeDraftGraphql
+  >('graphql', {
+    buildFields: [],
+    replaceFields: () => ({
```

Comment:
Maybe I'm missing something but here in the transformer we are replacing the `money` property of the draft model but I don't see that property in the generator so I don't understand where that property comes from and also why we are setting it to null here.

Could you maybe explain a bit this part? 🙏

Reply:
This is how it is modelled on GraphQL side. Since can't send an empty prop, has a dummy object inside. 

<img width="295" alt="Screenshot 2024-08-26 at 10 23 34" src="https://github.com/user-attachments/assets/951f46af-8f90-428c-a2b3-bae65e97806c">

<img width="304" alt="Screenshot 2024-08-26 at 10 23 38" src="https://github.com/user-attachments/assets/f4dfa260-8998-4f99-a059-fe65db762a4d">

Reply:
Ok. After some digging in the docs and the GraphQL explorer I think I get it.